src/client.js

```diff
@@ -157,6 +157,7 @@ spat.updateMeta = () => {
     { query: 'meta[property="og:type"]', key: 'content', value: head.ogp.type },
     { query: 'meta[property="og:image"]', key: 'content', value: head.ogp.image },
     { query: 'link[rel="canonical"]', key: 'href', value: head.canonical },
+    { query: 'link[rel="icon"]', key: 'href', value: head.favicon },
```

Comment:
@cigar-jp rel="shortcut icon" だからこれで

```suggestion
    { query: 'link[rel~="icon"]', key: 'href', value: head.favicon },
```

Reply:
@simiraaaa 
ありがとうございます、修正しました！
8602482